docs/overview/script-storage.md

```diff
@@ -0,0 +1,33 @@
+---
+title: 'Office Scripts file storage and ownership'
+description: 'Information about how Office Scripts are stored in OneDrive and transferred between owners.'
+ms.date: 11/13/2020
+localization_priority: Normal
+---
+
+# Office Scripts file storage and ownership
+
+Office Scripts are stored as **.osts** files in your OneDrive. The permissions of a script inside a workbook are limited to the permissions of that script's owner. The script **.osts** files carry whatever shared access and permissions that user's OneDrive allows.
+
+## File storage
+
+You Office Scripts are stored in your OneDrive. The **.osts** files are found in the **/Documents/Office Scripts/** folder. Any edits made these **.osts** files, such as renaming or deleting files, will be reflected in the Code Editor and Script Gallery.
+
+Scripts that are shared with one of your workbooks remain in the cloud and are not locally copied. The **Make a Copy** button of the Code Editor saves a separate, local copy of the script.Changes to the copy do not affect the original script.
```

Comment:
Space needed before "Changes to the copy..

Reply:
We should say Make a copy button of the Code Editor saves a separate copy of the script stored in the user's OneDrive (it isn't local).